test/WebJobs.Script.Tests.Shared/TestLogger.cs

```diff
@@ -74,14 +72,19 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
                 Exception = exception,
                 FormattedMessage = formatter(state, exception),
                 Category = Category,
-                Timestamp = DateTime.UtcNow
+                Timestamp = DateTime.UtcNow,
+                HostInstanceId = HostInstanceId
             };
 
             lock (_syncLock)
             {
                 _logMessages.Add(logMessage);
             }
-            _testOutput?.WriteLine($"{logLevel}: {formatter(state, exception)}");
+
+            _testOutput?.WriteLine($"{logMessage}");
+
+            // Uncomment this when debugging test failures in CI.
+            // Console.WriteLine($"{logMessage}");
```

Comment:
I'm leaving this here for others. If you uncomment this in a PR/branch and run in CI, you'll get all of the logs written to the Job's output. This lets you see things happening in the background, etc, whether or not they were captured explicitly in the test's output.

Reply:
This is awesome, thank you!